.kokoro/samples/python3.9/periodic-head.cfg

```diff
@@ -0,0 +1,11 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+env_vars: {
+    key: "INSTALL_LIBRARY_FROM_SOURCE"
+    value: "True"
+}
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/python-pubsub/.kokoro/test-samples-against-head.sh"
```

Comment:
```suggestion
    value: "github/python-aiplatform/.kokoro/test-samples-against-head.sh"
```

Reply:
Fixed in https://github.com/googleapis/python-aiplatform/pull/175/commits/437b5a138ef135353b08d12ae1a217924d4cacfe